packages/widgets-internal/ifo/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { ChainId } from "@pancakeswap/chains";
+
+// TODO: Consider moving to IFO package
+export const ChainNameMap = {
+  [ChainId.BSC]: "BSC",
```

Comment:
i think we already have a similar mapping exported from the chains package.

Reply:
if not lets actually add one

Reply:
Yeah I was wondering why this didn't exist. Right now a bit tight on timeline so maybe this can be a separate PR? @ChefBingbong

Reply:
yeah of course we can. do this another time for sure